internal/entities/rpc.go

```diff
@@ -31,6 +31,7 @@ type RPCGetTransactionResult struct {
 	OldestLedger          int64     `json:"oldestLedger"`
 	OldestLedgerCloseTime string    `json:"oldestLedgerCloseTime"`
 	ApplicationOrder      int64     `json:"applicationOrder"`
+	Hash                  string    `json:"txHash"`
```

Comment:
@gouthamp-stellar Do we have any unittests. That way we can ensure all such future changes are working fine.